mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -223,12 +238,16 @@ class LocalLDAModel private[clustering] (
    *
    * @param documents test corpus to use for calculating log likelihood
    * @return variational lower bound on the log likelihood of the entire corpus
+   * @since 1.3.0
```

Comment:
According to my research , it is 1.5.0. This methid was commented out in at least 1.3.0 and 1.4.0. So it might be grepped.